dtfs-central-api/api-tests/v1/gef-deals/add-min-activities.api-test.js

```diff
@@ -0,0 +1,413 @@
+    const expectedDate = format(new Date(), 'dd-MMMM-yyyy');
+    expect(receivedDate).toEqual(expectedDate);
+
+    expect(portalActivityObject.text).toEqual('');
+
+    expect(portalActivityObject.label).toEqual(PORTAL_ACTIVITY_LABEL.FACILITY_CHANGED_ISSUED);
+
+    // author should be undefined
+    const author = {
+      firstName: undefined,
+      lastName: undefined,
+      _id: undefined,
+    };
+    expect(portalActivityObject.author).toEqual(author);
+
+    // checker object - checks matches test checker
+    const checkerInObject = {
+      firstname: checker.firstname,
+      surname: checker.surname,
+      _id: JSON.stringify(checker._id),
+    };
+
+    expect(portalActivityObject.checker.firstname).toEqual(checkerInObject.firstname);
+    expect(portalActivityObject.checker.surname).toEqual(checkerInObject.surname);
+    expect(JSON.stringify(portalActivityObject.checker._id)).toEqual(checkerInObject._id);
+
+    // maker object matches the unissuedToIssuedByMaker
+    const makerInObject = {
+      firstname: mockFacilityOne[0].unissuedToIssuedByMaker.firstname,
+      surname: mockFacilityOne[0].unissuedToIssuedByMaker.surname,
+      _id: mockFacilityOne[0].unissuedToIssuedByMaker._id,
+    };
+
+    expect(portalActivityObject.maker).toEqual(makerInObject);
+
+    expect(portalActivityObject.html).toEqual('facility');
+
+    expect(portalActivityObject.facilityType).toEqual(`${mockFacilityOne[0].type} facility`);
+
+    expect(portalActivityObject.facilityID).toEqual(mockFacilityOne[0].ukefFacilityId);
+  });
+
+  it('should return a populated array with 2 in the issued facility activity object', async () => {
+    await wipeDB.wipe([collectionName]);
+    await wipeDB.wipe([applicationCollectionName]);
+
+    // resets array to length 0
+    MOCK_APPLICATION_FACILITIES.portalActivities = [];
+
+    const userCollection = await db.getCollection('users');
+
+    const checker = await userCollection.findOne({ roles: ['checker'] });
+    const maker = await userCollection.findOne({ roles: ['maker'] });
+
+    MOCK_APPLICATION_FACILITIES.checkerId = checker._id;
+
+    const makerToInsert = {
+      firstname: maker.firstname,
+      surname: maker.surname,
+      _id: maker._id,
+      email: maker.email,
+      roles: maker.roles,
+    };
+    MOCK_APPLICATION_FACILITIES.maker = makerToInsert;
+
+    const result = await facilityChangePortalActivity(MOCK_APPLICATION_FACILITIES, mockFacilitiesArray);
+
+    // ensure that only 1 object added to empty array
+    expect(result.length).toEqual(2);
+
+    // check that the first object is position 1 in mockFacilitiesArray
+    const portalActivityObjectZero = result[0];
+    expect(portalActivityObjectZero.facilityID).toEqual(mockFacilitiesArray[1].ukefFacilityId);
+
+    // check that second object is position 0 in mockFacilitiesArray
+    const portalActivityObjectOne = result[1];
+    expect(portalActivityObjectOne.facilityID).toEqual(mockFacilitiesArray[0].ukefFacilityId);
```

Comment:
I've added this in a different file as well, but can we change it to `facilityId` ?